index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <h1 class="header">Hello,</h1>
+</body>
```

Comment:
- [ ] You correctly created the CSS file, but the color is not being added to the 'Hello' text because it is not being imported by the HTML file. Kindly import the styles from the CSS file in the `head` of the `index.html` file and make sure that the styling is being applied.